### DISCUSSION TITLE
想要开发一个组件库文档，选用什么文档框架可以用来渲染Taro组件？

### DISCUSSION THREAD

**Discussion by AaronDaiR** [NervJS/taro#16332] [Q & A]
想要开发一个组件库文档，选用什么文档框架可以用来渲染Taro组件？市面上的stroybooks、dumi，开发组件库文档没什么经验，看了下都只支持react，还是我太菜，不太明白，求指点。

**Comment by AaronDaiR** [comment]
有没有人啊，有没有好心人呀，有没有大神啊~~

**Comment by AaronDaiR** [comment]
@honlyHuang @Leechael @guotie @Psli @smoothdvd

**Reply by honlyHuang** [reply]
你选的这两个都可以的哈，你编译成 H5 可以参考一下 NutUI 或者 TaroUI
